pkg/jobs/trigger_at.go

```diff
@@ -0,0 +1,89 @@
+package jobs
+
+import (
+	"time"
+
+	"github.com/cozy/cozy-stack/web/jsonapi"
+)
+
+// AtTrigger implements the @at trigger type. It schedules a job at a specified
+// time in the future.
+type AtTrigger struct {
+	typ  string
```

Comment:
Oh indeed. I did not think of that. done d6e801db5e104d8973e9f212eda49781bc8a67cf